.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
-sudo: false
+sudo: required
```

Comment:
This is no longer necessary and the `sudo` keyword is deprecated: https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration

Reply:
Applied. Thanks!